openapi.yml

```diff
@@ -0,0 +1,506 @@
+          schema:
+            $ref: '#/components/schemas/Druid'
+        - name: workflow
+          in: path
+          required: true
+          example: accessionWF
+          schema:
+            $ref: '#/components/schemas/Workflow'  
+    post:
+      summary: Add provided workflow to object
+      responses:
+        '200':
+          description: Request succeeded
+      parameters:
+        - name: druid
+          in: path
+          required: true
+          example: druid:bc123df4567
+          schema:
+            $ref: '#/components/schemas/Druid'
+        - name: workflow
+          in: path
+          required: true
+          example: accessionWF
+          schema:
+            $ref: '#/components/schemas/Workflow'  
+  /objects/{druid}/workflows/{workflow}/{process}:
+    put:
+      summary: Add a process to the provided workflow and druid
+      responses:
+        '200':
+          description: Request succeeded
+      parameters:
+        - name: druid
+          in: path
+          required: true
+          example: druid:bc123df4567
+          schema:
+            $ref: '#/components/schemas/Druid'
+        - name: workflow
+          in: path
+          required: true
+          example: accessionWF
+          schema:
+            $ref: '#/components/schemas/Workflow'  
+        - name: process
+          in: path
+          required: true
+          example: publish
+          schema:
+            $ref: '#/components/schemas/Process'
+  /workflow_templates:
+    get:
+      summary: Retrieve workflow template for given workflow
+      responses:
+        '200':
+          description: Workflow datastreams found
+          content:
+            application/json:
+              schema:
+                type: string
+  /workflow_templates/{workflow}:
+    get:
+      summary: Retrieve workflow template for given workflow
+      responses:
+        '200':
+          description: Workflow datastream found
+          content:
+            application/json:
+              schema:
+                type: string
+      parameters:
+        - name: workflow
+          in: path
+          required: true
+          example: accessionWF
+          schema:
+            $ref: '#/components/schemas/Workflow'
+  /workflow_archive:
+    get:
+      summary: Retrieve archived workflows
+      responses:
+        '200':
+          description: Request succeeded
+          content:
+            application/xml:
+              schema:
+                type: string
+      parameters:
+        - name: workflow
+          in: query
+          required: false
+          example: accessionWF
+          schema:
+            $ref: '#/components/schemas/Workflow'
+  /workflow_queue:
+    get:
+      summary: Retrieve the workflow queue
+      responses:
+        '200':
+          description: Request succeeded
+          content:
+            application/xml:
+              schema:
+                type: string
+      parameters:
```

Comment:
There may be some missing parameters here?

* `lane-id`
* `workflow`

Reply:
done.